src/app/shared/temporary-consts.ts

```diff
@@ -0,0 +1,28 @@
+import { Prefix, Option } from './models/temporary-models';
+
+export const PHONE_NUMBER_REGEX = /^\d{9}$/;
+
+export const prefixes: Prefix[] = [
+  {
+    countryCode: 'pl_PL',
```

Comment:
mamy enum LanguageCode

Reply:
nie wiem czy będziemy brać to z backendu do MVP 1, ja bym raczej szedł w kierunku tego, że będzie zmockowane na prod i włożył ten plik gdzieś do osobnego pliku ;)